sdk/src/wallet/core/builder.rs

```diff
@@ -203,56 +250,88 @@ where
         // It happened that inputs got locked, the transaction failed, but they weren't unlocked again, so we do this
```

Comment:
comment could be better

Reply:
Let's keep it for now, there will 100% be a refactoring of this file soon, we'll clean up the comments then